test/jdk/java/io/ObjectInputStream/PeekInputStreamTest.java

```diff
@@ -33,6 +33,9 @@
  * @modules java.base/java.io:open
  * @summary verifies java.io.ObjectInputStream.PeekInputStream.skip works
  *          as intended
+ *
+ * @run main PeekInputStreamTest
```

Comment:
We should update legal statement since we changed the test files.

Reply:
done